src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -428,6 +457,19 @@ private function _process_directives( string $html ) {
 				'exit'  => $p->is_tag_closer() || ! $p->has_and_visits_its_closer_tag(),
 			);
 
+			// Get the element attributes to include them in the element representation.
+			$element_attrs = array();
+			$attr_names    = $p->get_attribute_names_with_prefix( '' ) ?? array();
```

Comment:
Hmm... I think I remember hearing Dennis say that this is an expensive operation so maybe we should do it only when the `get_element` function is called, instead of by default for all the tags. 

Let's wait until Dennis returns from his sabbatical to confirm it, and if that's the case, we can include it in an additional pull request.